samples/nodejs_simple_sample/src/gateway_sample_lin.json

```diff
@@ -25,23 +34,23 @@
             "loader": {
                 "name": "node",
                 "entrypoint": {
-                    "main.path": "<<path to iothub_writer.js>>"
+                    "main.path": "../../../../samples/nodejs_simple_sample/nodejs_modules/iothub_writer.js"
                 }
             },
             "args": {
-                "connection_string": "<<IoT Hub Device Connection String>>"
+                "connection_string": "<IoTHub device connection string>"
```

Comment:
It should be IoT Hub...that's the official name (you'll see them squished together in variable names)